benchalerts/parse_conbench.py

```diff
@@ -46,21 +80,36 @@ def benchmarks_with_z_regressions(
 def regression_summary(
     comparisons: List[RunComparison], warn_if_baseline_isnt_parent: bool
 ) -> str:
-    """Generate a Markdown summary of what happened regarding regressions."""
+    """Generate a Markdown summary of what happened regarding errors and regressions."""
     sha = comparisons[0].contender_info["commit"]["sha"][:8]
+    errors = benchmarks_with_errors(comparisons)
+    regressions = benchmarks_with_z_regressions(comparisons)
+    summary = ""
+
+    if errors:
+        summary += "# Benchmark errors"
+        previous_run_id = ""
```

Comment:
Do we want to have a kind of summary or explanation here? Something like "These are errors that were caught while running the benchmarks. You can click the link next to each to go to the conbench entry for that benchmark which might have more information about what the error was. The data reported for them (if there is any) is suspect and shouldn't generally be compared with when the benchmarks succeeded"

Maybe even dropping that last sentence — it's important but I might be worried about something that's not a big deal (and we would need to also record times along with errors which we don't yet do...)

Reply:
Yeah I think we actually can't report times with errors yet, so I'll leave that last sentence out for now